activemodel/lib/active_model/dirty.rb

```diff
@@ -174,12 +174,10 @@ def changed_attributes
     end
 
     # Handles <tt>*_changed?</tt> for +method_missing+.
-    def attribute_changed?(attr, options = nil) #:nodoc:
+    def attribute_changed?(attr, from: :__none__, to: :__none__) # :nodoc:
```

Comment:
Can we use a poison object here instead so that all possible user provided values are valid?